### DISCUSSION TITLE
Scroll text

### DISCUSSION THREAD

**Discussion by diegorandal** [lovyan03/LovyanGFX#400] [Q&A] [answered]
Thank you Lovyan for this fabulous library.

I haven't found a way to make text scrolling work. In the library example, without changing anything, it doesn't seem to work either.

Any advice or idea?

**Comment by lovyan03** [comment, marked as answer]
Text scrolling is accomplished by the copy rectangle function.
This means that it will not work without a functioning pixel readout from the LCD.
It must be wired and configured correctly, using a panel with functioning pixel readout.
